### ISSUE TITLE
Update the generated clients

### ISSUE THREAD

**Issue by macnibblet** [paralin/go-dota2#3]
I attempted to use the tools in the repo to update the generated client code & protobuf without much success. Any chance you could look into it?

**Comment by paralin** [comment]
Yeah - I looked at it, it's all busted. Needs a lot of work to fix. Will look at it again when I have time.
